vnext/PropertySheets/Bundle.Common.targets

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+  Copyright (c) Microsoft Corporation. All rights reserved.
+ Licensed under the MIT License.. 
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildThisFileDirectory)\Bundle.props"/>
+
+  <Target Name="ReactNativeWindowsMakeBundle" BeforeTargets="PrepareForBuild" 
+      Condition="'$(UseBundle)' == 'true'">
+        <MakeDir Directories="$(BundleContentRoot)" Condition=" !Exists('$(BundleContentRoot') and '$(BundleContentRoot)' != '' " />
+        <Warning Text="BundleCommand is not defined, and you should execute 'react-native bundle' by yourself" Condition="'$(BundleCommand)' == ''" />
+        <Exec Condition="'$(BundleCommand)' != ''" Command="$(BundleCommand)" ConsoleToMSBuild="true" WorkingDirectory="$(SolutionDir).."/>
```

Comment:
To prevent the `..` from showing up in the logs, use this expression:
```
WorkingDirectory="$([System.IO.Path]::GetDirectoryName($(SolutionDir)))"
```

Reply:
Fixed